docs/self-managed/identity/what-is-identity.md

```diff
@@ -5,16 +5,12 @@ sidebar_label: "What is Identity?"
 description: "Identity is the component within the Camunda 8 stack responsible for authentication and authorization."
 ---
 
-:::note Looking for IAM documentation?
-From version 8.0.0+ the IAM component has been replaced with the Identity component.
-:::
-
 Identity is the component within the Camunda 8 stack responsible for authentication and authorization. It allows you to manage:
 
-- Applications
-- APIs
-- Permissions
-- Roles
+- [Applications](/self-managed/concepts/access-control/applications.md)
+- [APIs](/self-managed/concepts/access-control/apis.md)
+- [Permissions](/self-managed/concepts/access-control/resource-authorizations.md)
+- [Roles](/self-managed/concepts/access-control/resource-authorizations.md)
```

Comment:
Permissions and role are handled via the same docs page in this concepts section, right?

Reply:
So we have [add an assign a **role** to a user](https://docs.camunda.io/docs/next/self-managed/identity/user-guide/roles/add-assign-role/), and [add and assign a **permission** to a role](https://docs.camunda.io/docs/next/self-managed/identity/user-guide/roles/add-assign-permission/). I would assume users would create roles _before_ creating permissions for such roles, so regardless the two tabs should likely be flipped here in more of a chronological order.

Reply:
Good idea! I like your idea.